src/context/TodoProvider.js

```diff
@@ -0,0 +1,51 @@
+import { createContext, useContext, useState } from "react";
+
+const TodoContext = createContext({
+  todos: [],
+  addNewTodo: () => null,
+  deleteTodo: () => null,
+  achieveTodo: () => null,
+});
+
+export const useTodo = () => useContext(TodoContext);
+
+const TodoProvider = ({ children }) => {
+  const [todos, setTodos] = useState(
+    JSON.parse(localStorage.getItem("todoscontext")) || []
```

Comment:
You could actually consider using the `camelCase` naming convention for all the `localStorage` item names.